### PR TITLE
Fix ALRU wake up time test and documentation

### PR DESCRIPTION
**Pull request by arutk**

### CODE REVIEW

**Review by robertbaldyga — APPROVED** [review]

**Comment by arutk** [comment]
retest this please

**Review by mmichal10 — APPROVED** [review]

**Review by jfckm — APPROVED** [review]

File: casadm/casadm.8 [hunk 1]
```diff
@@ -286,7 +286,9 @@ Identifier of cache instance <1-16384>.
 
 .TP
 .B -w, --wake-up <NUMBER>
-Period of time between awakenings of flushing thread [s] (default: 20 s).
+Cleaning thread sleep time after an idle wake up [s] (default: 20 s).
+Idle wake up happens when there is no dirty data or the cleaning thread does not start cleaning
+due to stalenes time and/or activity threshold constraints.
```
Comment (robertbaldyga) [inline casadm/casadm.8:291]: Aspell test fails on this: s/stalenes/staleness

**Review by robertbaldyga — CHANGES_REQUESTED** [review]

File: test/functional/tests/lazy_writes/cleaning_policy/test_cleaning_params.py [hunk 2]
```diff
@@ -141,10 +141,13 @@ def test_cleaning_policy_config():
...
             )
-        elif core_dirty_before != core_dirty_after + data_to_flush:
+        # Only check whether a minimum amount of data is flushed, as ALRU does not have
+        # a configurable sleep time between active flushing iterations which would allow
+        # to precisely estimate expected amount of data.
+        elif core_dirty_before < core_dirty_after + data_to_flush:
             TestRun.LOGGER.error(
                 f"Number of dirty blocks flushed differs from configured in policy.\n"
-                f"Expected dirty data flushed: {data_to_flush}\n"
+                f"Expected minumum dirty data flushed: {data_to_flush}\n"
```
Comment (rafalste) [inline test/functional/tests/lazy_writes/cleaning_policy/test_cleaning_params.py:150]: and a small typo

Comment (arutk) [reply test/functional/tests/lazy_writes/cleaning_policy/test_cleaning_params.py]: fixed

**Review by robertbaldyga — APPROVED** [review]

**Pull request merged**